core/src/main/scala/org/apache/spark/broadcast/BroadcastManager.scala

```diff
@@ -58,8 +59,19 @@ private[spark] class BroadcastManager(
     new ReferenceMap(AbstractReferenceMap.HARD, AbstractReferenceMap.WEAK)
   }
 
+  private def attachBroadcastIdForPython[T: ClassTag](value_ : T, bid: Long): Unit = {
```

Comment:
and can we add some comments, as a proper method doc, here about how it works? TBH, looks kind of error-prone.